chart/kubenab/Chart.yaml

```diff
@@ -1,9 +1,8 @@
 apiVersion: v1
 name: kubenab
-version: 0.0.8
+version: 0.0.9
 appVersion: 0.3.3
 home: https://github.com/jfrog/kubenab
-kubeVersion: ">=1.12.0"
```

Comment:
`kubeVersion` _should_ be defined to have a required minimum Version for K8s

Reply:
I removed it because it clashes with EKS v1.14.7-eks-1861c5. I now see that it's possible to override kubeversion using `helm --kube-version` so this is not required.

Reply:
What is not required?